src/data/repositories/UserD2ApiRepository.ts

```diff
@@ -39,6 +46,7 @@ export class UserD2ApiRepository implements UserRepository {
                 pageSize,
                 query: search !== "" ? search : undefined,
                 filter: otherFilters,
+                rootJunction: filters ? rootJunction : undefined,
                 order: `${sorting.field}:${sorting.order}`,
             })
```

Comment:
I guess this problem is know, but I noticed the filter "Display only users I can manage" does nothing. The old code passed the param "canManage=true", but it's not in the new code.

Reply:
That's very interesting, I certainly didn't know it, and I was a bit puzzled about what that filter meant (and still am).

I checked and it seems that in `UserD2ApiRepository.ts` there has never been a `canManage` (nor `showOnlyManagedUsers` or anything like that). Maybe the bug came when this part was refactored to follow a clean architecture?

In any case, I think that should be filled as a (separate) bug/task.